packages/react-components/src/components/ParticipantList.tsx

```diff
@@ -102,6 +102,7 @@ const onRenderParticipantsDefault = (
  */
 export const ParticipantList = (props: ParticipantListProps): JSX.Element => {
   const { excludeMe = false, myUserId, participants, onParticipantRemove, onRenderAvatar, onRenderParticipant } = props;
+  const ids = useIdentifiers();
```

Comment:
Does this mean that all uses of `ParticipantList` *must* be wrapped with an `IdentifierProvider` now?

We want to avoid that. Providers are a nifty tool to avoid drilling through props. But they shouldn't replace props in the component definition.

This is related to my comment about removing the `IdentifierProvider` from the API.

Reply:
Nope. Think of providers as a way to set a context. If the context is already there, this hook will return the default values even when the component is not wrapped inside provider.